notebooks/index.ipynb

```diff
@@ -6,36 +6,31 @@
    "source": [
```

Comment:
I must have done something wrong with using ReviewNB as it didn't mark my comments as requesting changes, this comment should cause that